lib/cli/interactive-setup/service.js

```diff
@@ -32,12 +32,15 @@ const initializeProjectChoices = [
   { name: 'Other', value: 'other' },
 ];
 
-const projectTypeChoice = async (stepHistory) => {
+const projectTypeChoice = async ({ stepHistory, options }) => {
   const projectType = await promptWithHistory({
     message: 'What do you want to make?',
     type: 'list',
     name: 'projectType',
-    choices: initializeProjectChoices,
+    choices: initializeProjectChoices.filter(
+      // Temporarily filter out Python and SQS projects when console support is intended
+      ({ value }) => !options.console || (!value.includes('python') && !value.includes('sqs'))
```

Comment:
what about the starter templates that don't have any events? Are these supported by console? Last time I was checking these functions' invocations weren't showing up (only API)

Reply:
Technically all Node.js lambda invocations are reported. If they do not appear in Console, I'd take it's an issue that either was already handled or will be handled shortly